extensions/console/01_colors.js

```diff
@@ -55,10 +55,6 @@
     return run(str, code(35, 39));
   }
 
-  function dim(str) {
-    return run(str, code(2, 22));
-  }
```

Comment:
I think it's better that we just don't use dim since it doesn't work in cmd and instead use gray. I've removed this, but let me know if I should re-add it.